web/libs/editor/src/tags/object/RichText/RichText.scss

```diff
@@ -2,9 +2,9 @@
   position: relative;
 
   &__container {
-    overflow: auto;
+    // overflow: auto;
```

Comment:
Should be removed if not needed

Reply:
rewrote it with FF just in case, because it might change the overall experience with Text tag. thank you!